opentelemetry-otlp/Cargo.toml

```diff
@@ -15,10 +15,16 @@ keywords = ["opentelemetry", "otlp", "logging", "tracing", "metrics"]
 license = "Apache-2.0"
 edition = "2018"
 build = "build.rs"
+autotests = false
 
 [lib]
 doctest = false
 
+[[test]]
+name = "smoke"
+path = "tests/smoke.rs"
+required-features = ["integration-testing"]
```

Comment:
Instead of this, we could just add `#![cfg(feature = "integration-testing")]` to the test, any thoughts on which is better?

Reply:
i guess if you are ok with the conditionally public proto mod, i'd leave it like this. this is where integration tests belong.